src/containers/Body/models.ts

```diff
@@ -12,8 +12,10 @@ export namespace Payload {
 
 export namespace Action {
   export type SetData = Models.IAction<Payload.SetData>;
+  export type SetStartData = { type: string };
```

Comment:
`export type SetStartData = Models.IAction<void>;`

Or to do something like this:
```ts
export interface IEmptyAction {
  type: string;
}

export interface IAction<T> extends IEmptyAction {
  payload: T;
}
```

```ts
export type SetStartData = Models.IEmptyAction;
```